cmd/skaffold/app/cmd/build.go

```diff
@@ -100,3 +95,18 @@ func runBuild(out io.Writer) error {
 
 	return nil
 }
+
+func createRunnerAndBuild(ctx context.Context, buildOut io.Writer) ([]build.Artifact, error) {
+	runner, config, err := newRunner(opts)
+	var targetArtifacts []*latest.Artifact
+	for _, artifact := range config.Build.Artifacts {
+		if runner.IsTargetImage(artifact) {
+			targetArtifacts = append(targetArtifacts, artifact)
+		}
+	}
+	if err != nil {
+		return nil, errors.Wrap(err, "creating runner")
+	}
+	defer runner.RPCServerShutdown()
```

Comment:
Shouldn't it be called just after `runner, config, err := newRunner(opts)`

Reply:
yeah i think, i moved code over by mistake.